pom.xml

```diff
@@ -377,6 +378,41 @@
         </executions>
       </plugin>
 
+      <!-- Flatten pom plugin -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>flatten-maven-plugin</artifactId>
+        <version>${plugin.version.flatten}</version>
+        <configuration combine.children="append">
+          <flattenMode>ossrh</flattenMode>
```

Comment:
🔧 The `ossrh` mode of the flatten plugin removes unnecessary tags and inlines properties and relationships. It highlights some inherited tags from the parent of this pom `camunda-release-parent`. Please overwrite the `description`:
- in the root pom as empty
- or, in the root pom as a description of the root pom + in all other poms with a description of that project.

💭 Also note that the `developers` section is currently set to:

```xml
  <developers>
    <developer>
      <id>camunda.org</id>
      <name>camunda BPM Community</name>
      <organization>camunda.org</organization>
      <organizationUrl>http://camunda.org</organizationUrl>
    </developer>
  </developers>
```

This is the same as zeebe, but I'm not sure whether it's actually correct there. So I'm also not sure about it for this project. Please give it some thought.